packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/ios/redirecturi.md

```diff
@@ -0,0 +1,3 @@
+If you are setting up the sample app, use this value:
```

Comment:
I'm not sure how clear this is to a developer.  Setting up the sample app can mean different things; perhaps it might be worthwhile to explicitly call out the app in the Okta admin dashboard.